rskj-core/src/main/java/co/rsk/core/RskAddress.java

```diff
@@ -116,4 +116,8 @@ public int hashCode() {
     public String toString() {
         return Hex.toHexString(bytes);
     }
+
+    public int length() {
```

Comment:
I think it's better to not expose this method and use the length of the `co.rsk.core.RskAddress#getBytes` array. This is because there is an special address in our domain, which is the Remasc sender, who has length equals to 1. It shouldn't be the case for this new opcode to be used by that address, but for keeping consistency along all possible RskAddress in our domain it's better to get the address length in the way I mentioned before.